packages/gatsby-plugin-manifest/README.md

```diff
@@ -69,6 +69,10 @@ There are three modes in which icon generation can function: automatic, hybrid,
   - Legacy icon support - yes
   - Cache busting - never
 
+- i18n - Generate separate versions for multiple languages
```

Comment:
Should be listed as a feature under each mode.

Reply:
Like
```
- Cache busting - yes
- i18n - yes
```
or better `internationalization`?

Reply:
Maybe `- internationalization - optional` Cause `yes` implies it's the default behavior (or at least that was my intent when I wrote this).